config/config.go

```diff
@@ -68,6 +68,8 @@ func NewConfig() (*Config, []cli.Flag) {
 		hardCodedEnv: []string{
 			"NETFLIX_APPUSER=appuser",
 			"EC2_DOMAIN=amazonaws.com",
+			"AWS_METADATA_SERVICE_TIMEOUT=5",
```

Comment:
Maybe add a comment that these are standard AWS SDK environment vars?  Other than that, LGTM.